src/analysis/lattice.h

```diff
@@ -42,15 +43,31 @@ constexpr bool is_lattice = has_getBottom<Lattice>&& has_compare<Lattice>&&
 // Represents a powerset lattice which is constructed from a finite set which
 // can be represented by a bitvector. Set elements are represented by
 // FinitePowersetLattice::Element, which represents members present in each
-// element by bits in the bitvector.
-class FinitePowersetLattice {
+// element by bits in the bitvector. Type is the type of a member in a lattice
+// element.
+template<typename Type> class FinitePowersetLattice {
 
   // The size of the set that the powerset lattice was created from. This is
   // equivalent to the size of the Top lattice element.
   size_t setSize;
```

Comment:
We probably don't need to continue storing `setSize` separately if we can now just use `memberOrdering.size()`.

Reply:
`setSize` is still needed for `FiniteIntPowersetLattice` (the plain bitvector class). However, it it doesn't need to be additionally instantiated for `FinitePowersetLattice`.